frontend/src/components/modals/editName/EditAccountModal.tsx

```diff
@@ -0,0 +1,72 @@
+import { FileUploader } from "react-drag-drop-files";
+
+import {
+    EditAccountWrapper,
+    OptionSeparator,
+    OptionEditAccount,
+    TextInput,
+    OptionEditAvatar
+} from "./EditAccountModal.styled";
+import { ModalSpan } from "../Modals.styled";
+import { PrimaryButton } from "../../buttons/Button.styled";
+
+import useModalEditUsername from "./useModalEditUsername";
+import useModalEditAvatar from "./useModalEditAvatar";
+
+import { IModal } from "../modals.interfaces";
+
+const EditAccountModal = ({ hideModal }: IModal) => {
+    const { newUsername, handleNicknameChange, updateUsername } =
+        useModalEditUsername(hideModal);
+
+    const {
+        fileTypes,
+        file,
+        setFile,
+        handleChange,
+        onTypeError,
+        onSizeError,
+        onDrop,
+        onSelect,
+        onDraggingStateChange
+    } = useModalEditAvatar();
+
+    return (
+        <>
+            <EditAccountWrapper>
+                <OptionEditAccount>
+                    <ModalSpan>Twój nowy nick:</ModalSpan>
+                    <TextInput
+                        value={newUsername}
+                        onChange={handleNicknameChange}
+                    />
+                    <PrimaryButton onClick={updateUsername}>
+                        Zapisz
+                    </PrimaryButton>
+                </OptionEditAccount>
+                <OptionSeparator />
```

Comment:
U think to use 'before' and 'after' operators in css, instead of new SC?

Reply:
refactored to border-bottom